docs/interface_lang_management.md

```diff
@@ -0,0 +1,56 @@
+<!--
+* Copyright (c) 2017, 2020 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Language management interface
+
+Eclipse OpenJ9 provides MXBean extensions to the standard `java.lang.management` API, which can be used to monitor and manage the Java@reg; virtual machine.
+These extensions provide access to information about the state of the OpenJ9 VM and the environment in which it is running. The following
+table lists the MXBeans and describes the monitoring or management capability:
+
+
+| MXBean                    | Description                                                                                                  |
+|---------------------------|--------------------------------------------------------------------------------------------------------------|
+| `GarbageCollectorMXBean`    | Discovers Garbage Collection (GC) operations (collection times, compactions, heap memory usage, and freed memory). |
+| `MemoryMXBean`              | Discovers memory usage (minimum and maximum heap sizes, and shared class cache sizes).               |
+| `MemoryPoolMXBean`          | Discovers memory pool usage for specific GC policies.                                                         |
+| `ThreadMXBean`              | Discovers information about native thread IDs.                                                                |
+| `JvmCpuMonitorMXBean`       | Discovers CPU consumption by category (GC, JIT or Other threads).                                             |
+| `ProcessorMXBean`           | Discovers and limits the CPUs that a process has access to.                                                  |
```

Comment:
This one shouldn't be in the list. It's not a properly installed MXBean. It shouldn't be in the IBM doc either, I guess we forgot to remove it in the last update.

Reply:
Removed